### DISCUSSION TITLE
[SOLVED ]Help on QTable rowNumber refresh

### DISCUSSION THREAD

**Discussion by odranoelBR** [quasarframework/quasar#8737] [General - Components / Directives / etc] [answered]
Hi community!

I’m creating a Crud component based on QTable and sometimes can be Server side requests, and sometimes simple pagination.

When i do

`delete this.pagination.rowsNumber;`
even with

`:pagination.sync="pagination"`
Qtable dont switch back to simple pagination
I tried

`$refs.table.setPagination(this.pagination)`
but without luck, any suggestions ?

**Comment by metalsadman** [comment]
Try to re-tender your qtable after deleting the property using `key`.

**Comment by rstoenescu** [comment, marked as answer]
```js
const { rowsNumber, ...pagination } = this.pagination
this.pagination = pagination
```

New reference means it's guaranteed that Vue detects the change.

**Comment by metalsadman** [comment]
Unfortunately qtable still wont go out of serverside mode and continue to call request on pagination object change unless it's forced to re-render, hence my suggestion for a key change. https://codepen.io/metalsadman/pen/PoWmpLP?editors=1010

**Comment by odranoelBR** [comment]
Thank you guys! Both solutions work like charm 
@rstoenescu @metalsadman 💪

**Reply by metalsadman** [reply]
> Thank you guys! Both solutions work like charm
> @rstoenescu @metalsadman 💪

Np, you can mark your thread as answered.

**Reply by ontwikkelfabriek** [reply]
@odranoelBR  would you be so kind to mark this post as answered( gets a green status)
